lib/setup.js

```diff
@@ -131,19 +153,29 @@ function isSemver(s) {
  * @returns {Promise<string>} The directory SAM CLI is installed in.
  */
 // TODO: Support more platforms
-async function installUsingNativeInstaller(version) {
+async function installUsingNativeInstaller(inputVersion) {
   if (os.platform() !== "linux" || os.arch() !== "x64") {
     core.setFailed("Only Linux x86-64 is supported with use-installer: true");
     return "";
   }
 
   // Must be full semantic version; downloads version directly from GitHub
-  if (version && !isSemver(version)) {
+  if (inputVersion && !isSemver(inputVersion)) {
     core.setFailed("Version must be in the format x.y.z");
     return "";
   }
 
-  // TODO: If not set, check latest version and return from cache if exists
+  let version = inputVersion;
+  if (!version) {
+    const latestVersion = await getLatestReleaseTag("aws/aws-sam-cli");
+    // Must be full semantic version; downloads version directly from GitHub
+    if (latestVersion && !isSemver(latestVersion)) {
+      core.setFailed("Latest version found must be in the format x.y.z");
+      return "";
```

Comment:
Do we need to validate `latestVersion`? I thought it's always using the format.

Reply:
It should, just in case if the return value is terribly wrong, we don't break it.